app/views/shared/export/_plan_txt.erb

```diff
@@ -1,4 +1,4 @@
-<%= "#{@plan.title}" %>
+<%= "#{@plan.title}".gsub(/&amp;/,'&') %>
```

Comment:
Can we use sanitize here as well ?

Reply:
These have to do with the outputted text when downloading a .txt file of a plan.

I included a couple of pictures. Only `.gsub()` seems to fix the ampersand in this case. But maybe using both `sanitize()` and `.gsub()` would be best?

![Screenshot from 2023-10-16 11-47-40](https://github.com/portagenetwork/roadmap/assets/71047780/9063840b-0cd6-4cb4-bf3d-57d4962c1e98)
![Screenshot from 2023-10-16 11-48-34](https://github.com/portagenetwork/roadmap/assets/71047780/c12f4757-bf1d-4256-85e4-9c9c33eafc62)

Reply:
I did see that Zuru is for filenames

Have you considered using `Zaru.sanitize! ` ?

From their [documentation](https://github.com/madrobby/zaru)